website/docs/components/color-picker/examples/inputValidation.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import ColorPicker, { PaletteManager } from '@semcore/color-picker';
+
+const onChange = (v, e) => {
+  console.log('value', v);
```

Comment:
This is a bad example( Users don't use browser with dev console open

Reply:
can I change console.log to alert? (: